arrow-data-source/pom.xml

```diff
@@ -49,30 +49,8 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-common</artifactId>
-      <version>${hadoop.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>com.fasterxml.jackson.core</groupId>
-          <artifactId>jackson-core</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.fasterxml.jackson.core</groupId>
-          <artifactId>jackson-annotations</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.fasterxml.jackson.core</groupId>
-          <artifactId>jackson-databind</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.slf4j</groupId>
-          <artifactId>slf4j-log4j12</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>log4j</groupId>
-          <artifactId>log4j</artifactId>
-        </exclusion>
-      </exclusions>
+      <artifactId>hadoop-client</artifactId>
```

Comment:
not sure if this will break v2 @rui-mo

Reply:
If executions are required they were added to the parent pom https://github.com/oap-project/native-sql-engine/pull/307/files/7a39548a3dedc3c8f89127f4cac712a4c20864c6#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R145-R170

Reply:
And is there other reasons v2 is breaked?

Reply:
the issue are mostly due to spark pre-build binary with hadoop 3.2 - which packages some specific version of depdencies other than those in `hadoop-common-3.2.0`. will do a quick test on my env to verify
https://github.com/apache/spark/blob/v3.1.1/pom.xml#L172
https://github.com/apache/spark/blob/v3.1.1/pom.xml#L121

Reply:
ok, I will check on this.

Reply:
@zhouyuan it is ok on my env

Reply:
thanks!